generated/provider_dependencies.json

```diff
@@ -435,7 +435,7 @@
       "aiohttp>=3.9.2, <4",
       "apache-airflow-providers-common-sql>=1.10.0",
       "apache-airflow>=2.8.0",
-      "databricks-sql-connector>=2.0.0, <3.0.0, !=2.9.0",
+      "databricks-sql-connector>=2.0.0, !=2.9.0",
```

Comment:
You should use pre-commit to regenerate this file - currrently you have different specification in both.